tce/icdecoder_plugins/DefaultICDecoderPlugin.cc

```diff
@@ -30,6 +30,10 @@
  * @author Vinogradov Viacheslav(added Verilog generating) 2012 
  * @note rating: red
  */
+ 
+ /*
```

Comment:
Please mind the 78 row size limit and also this type of comment belongs to git commit log, not in the source code. There you'd best describe what you did and preferably in more detail, why these changes were needed.